toolkit/tools/pkg/imagecustomizerlib/liveosisobuilder.go

```diff
@@ -413,12 +413,6 @@ func (b *LiveOSIsoBuilder) updateGrubCfg(isoGrubCfgFileName string, pxeGrubCfgFi
 		return fmt.Errorf("failed to update the root kernel argument in the iso grub.cfg:\n%w", err)
 	}
 
-	inputContentString, err = updateSELinuxCommandLineHelperAll(inputContentString, imagecustomizerapi.SELinuxModeDisabled,
```

Comment:
Is this just no longer needed at all? A quick look shows this was the only place its being used.

Reply:
Corect. In the past, to get liveos iso working, we always disabled selinux.
Now that we can handle selinux, there's no need to disable it. Instead, whatever the user wants (enable or disabled) will stay intact in the generated liveos iso.